src/components/Modal/index.module.scss

```diff
@@ -29,6 +29,7 @@
     position: absolute;
     right: 33px;
     top: 33px;
+    z-index: 2;
```

Comment:
Is this needed?

Reply:
Yes required, the close icon on Modal wasn't responding without this.